source/tutorial/install-mongodb-on-os-x.txt

```diff
@@ -29,17 +30,32 @@ information.
 Install MongoDB Community Edition
 ---------------------------------
 
-Prerequisite
-~~~~~~~~~~~~
+.. _osx-prereq:
+
+Prerequisites
+~~~~~~~~~~~~~
+
+If you have the Homebrew ``brew`` package installed on your OSX host 
```

Comment:
From Allison's testing (I'll confirm when I get home), running any of these instructions w/ `brew` installed and/or mongodb tapped doesn't do anything catastrophic (other than updating brew).